From 62caa5edea39430fbd632246d537b8ac8f71522d Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Sat, 28 Apr 2012 17:03:43 -0300
Subject: [PATCH 17/40] =?UTF-8?q?4.2.7=20Hijos=20de=20Hodir:=20=C2=A1Se=20ll?=
 =?UTF-8?q?evaron=20a=20nuestros=20Hombres!?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixed Goblin prisoner's SAI to reset the cages at spawn
---
 .../2012_04_24_100_world_they_took_our_men.sql     |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_04_24_100_world_they_took_our_men.sql

diff --git a/sql/updates/world/reanemu/2012_04_24_100_world_they_took_our_men.sql b/sql/updates/world/reanemu/2012_04_24_100_world_they_took_our_men.sql
new file mode 100644
index 0000000..b4852d2
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_04_24_100_world_they_took_our_men.sql
@@ -0,0 +1 @@
+UPDATE `smart_scripts` SET `event_param2`=571, `event_param3`=67 WHERE `entryorguid`=29466 AND `source_type`=0 AND `id`=0;
-- 
1.7.8.msysgit.0

